code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm

```diff
@@ -194,12 +206,26 @@ Difficulty: Hard
 	else //just release a burst of power
 		INVOKE_ASYNC(src, PROC_REF(burst), get_turf(src))
 
+/mob/living/simple_animal/hostile/megafauna/hierophant/proc/easy_anti_cheese(mob/living/simple_animal/S)
+	if(enraged || get_dist(S, src) > 20)
+		return
+	for(var/mob/living/L in urange(20, src))
+		if(L.client)
+			enrage()
+			arena_trap(L, TRUE)
+			FindTarget(list(L), 1)
+			for(var/mob/living/simple_animal/hostile/megafauna/colossus/C in GLOB.mob_list)
+				UnregisterSignal(C, COMSIG_MOB_APPLY_DAMAGE)
+			break
+
 /mob/living/simple_animal/hostile/megafauna/hierophant/proc/blink_spam(blink_counter, target_slowness, cross_counter)
 	ranged_cooldown = world.time + max(5, major_attack_cooldown - anger_modifier * 0.75)
-	if(health < maxHealth * 0.5 && blink_counter > 1)
+	if((health < maxHealth * 0.5 || enraged) && blink_counter > 1)
```

Comment:
This is one of those places where it may require some parenthesis
```suggestion
	if(((health < maxHealth * 0.5) || enraged) && blink_counter > 1)
```

Reply:
done